localstack/services/elbv2/elbv2_listener.py

```diff
@@ -0,0 +1,15 @@
+import re
+import logging
+from localstack.utils.common import to_str, to_bytes
+from localstack.services.generic_proxy import ProxyListener
+from moto.elbv2 import urls
+
+LOG = logging.getLogger(__name__)
+
+
+class ProxyListenerELBV2(ProxyListener):
```

Comment:
As we're currently not applying any logic in the proxy listener, I guess we can simply remove this file (and remove the `listener` argument from the `register_plugin(..)` call above).